kebechet/managers/version/version.py

```diff
@@ -206,7 +208,8 @@ def _compute_changelog(repo: Repo, old_version: str, new_version: str,
             # can be in case of the very first release.
             old_version = repo.git.rev_list("HEAD", max_parents=0)
 
-        changelog = repo.git.log(f'{old_version}..HEAD', no_merges=True, format='* %s').splitlines()
+        changelog = repo.git.log(
+            f'{old_version}..HEAD', no_merges=True, format='* %s').splitlines()[:_MAX_CHANELOG_SIZE]
```

Comment:
I think it would be better to move this stripping logic directly to source management - to respective create issue/create PR methods to have this available all over the place.

Also, it would be nice to mention that the body was stripped in the body itself.

Reply:
ack

Reply:
@fridex the kebechet info seems to be more than 300 lines for example, but we display it without truncating it. So I feel like limiting it on the source-management would limit on other managers which don't involve ultrahook limitation..

Reply:
OK, sounds reasonable in that case. Is this PR still valid? (it was closed)

Reply:
> OK, sounds reasonable in that case. Is this PR still valid? (it was closed)

Yes, let's merge this, if we notice a bigger PR body elsewhere, I will move it to source-management :)

Reply:
Do you want me to move this stripping logic to `_construct_pr_body` method? and let the whole changelog be committed to the file?

Reply:
> and let the whole changelog be committed to the file?

That sounds like a good approach :+1:

Reply:
Thanks, I have changed it